service/ztests/curl-add-gzip.yaml

```diff
@@ -14,5 +14,5 @@ inputs:
 
 outputs:
   - name: stdout
-    regexp: |
-      {"kind":"api.CommitResponse","value":{"commit":"0x\w{40}","warnings":\[]}}
+    data: |
+      {"type":"Error","kind":"item does not exist","error":"missing: pool not found"}
```

Comment:
Non-nit: This shouldn't change. It probably did because the contents of pool.json changed and `zq -f text "cut value.pool.id" pool.json)` isn't printing the pool ID.

Reply:
Oops, I thought I fixed that one.

Reply:
I just realized I had fixed this in another branch.   I have too many branches stacked up... the weekend effect.